package.json

```diff
@@ -161,7 +161,9 @@
     "setupFiles": [
       "jest-canvas-mock"
     ],
-    "setupTestFrameworkScriptFile": "<rootDir>/tests/setup.js",
+    "setupFilesAfterEnv": [
+      "<rootDir>/tests/setup.js"
+    ],
```

Comment:
Not related: Use `setupFilesAfterEnv` instead of `setupTestFrameworkScriptFile` which has deprecation warning.